pyproject.toml

```
@@ -0,0 +1,64 @@
+[tool.poetry]
+name = "simple-settings"
+version = "1.1.0"
+description = "A simple way to manage your project settings."
+authors = ["Diego Garcia <drgarcia1986@gmail.com>"]
+license = "MIT"
+keywords=["django", "flask", "bottle", "tornado", "settings", "configuration", "conf"]
+classifiers=[
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: MIT License',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 3',
+]
+packages = [
+    { include = "simple_settings" },
+    { include = "strategies", from = "simple_settings" },
+    { include = "dynamic_settings", from = "simple_settings" },
+]
+repository = "https://github.com/drgarcia1986/simple-settings"
+
+[tool.poetry.urls]
+"Download" = "https://github.com/drgarcia1986/simple-settings/tarball/master"
+
+[tool.poetry.dependencies]
+python=">=3.6.1,<4.0"
+jsonpickle = { version = "2.0.0", optional = true }
+toml = { version = "0.10.2", optional = true }
+PyYAML = { version = "6.0", optional = true }
+SQLAlchemy = { version = "1.4.26", optional = true }
+pymemcache = { version = "3.5.0", optional = true }
+six = { version = "1.16.0", optional = true}
+redis = { version = "3.5.3", optional = true }
+boto3 = { version = "1.9.201", optional = true }
```

These dependencies here are hard-pinned, but now there is also poetry.lock which has a specific set of known-good dependency versions, which are good enough for running the tests.

Since projects that depend on `simple-settings` might themselves need to depend on these libraries, it makes sense to be a little more flexible with their specification, like:

```suggestion
jsonpickle = { version = "^2.0.0", optional = true }
toml = { version = "^0.10.2", optional = true }
PyYAML = { version = "^6.0", optional = true }
SQLAlchemy = { version = "^1.4.26", optional = true }
pymemcache = { version = "^3.5.0", optional = true }
six = { version = "^1.16.0", optional = true}
redis = { version = "^3.5.3", optional = true }
boto3 = { version = "^1.9.201", optional = true }
```

For reference, see:

https://python-poetry.org/docs/dependency-specification/#caret-requirements

On the other hand, the code above is a direct transliteration of the old `setup.py` code, so perhaps the flexibilization I'm proposing should be the subject of a different PR...

---

Indeed I sticked religiously to the versions in setup.py to minimize possible side-effects of bumping versions and thinking the issue is on Poetry side, but I agree with your suggestion. @drgarcia1986 any thoughts on bumping the versions in this PR or in a different one?

---

I prefer in a different one